modules/troubleshoot/partials/debug-bundle/generate-rpk.adoc

```diff
@@ -0,0 +1,266 @@
+
+```bash
+kubectl create clusterrolebinding redpanda --clusterrole=view --serviceaccount=redpanda:default
+```
+====
+endif::[]
+
+
+. Execute the xref:reference:rpk/rpk-debug/rpk-debug-bundle.adoc[`rpk debug bundle`] command on a broker:
++
+[,bash]
+----
+ifdef::env-kubernetes[]
+kubectl exec -it --namespace <namespace> redpanda-0 -c redpanda -- rpk debug bundle --namespace <namespace>
+endif::[]
+ifndef::env-kubernetes[]
+rpk debug bundle
+endif::[]
+----
++
+If you have an upload URL from the Redpanda support team,
+provide it in the `--upload-url` flag to upload your debug bundle to Redpanda.
++
+[,bash]
+----
+ifdef::env-kubernetes[]
+kubectl exec -it --namespace <namespace> redpanda-0 -c redpanda -- rpk debug bundle \
+  --upload-url <url> \
+  --namespace <namespace>
+endif::[]
+ifndef::env-kubernetes[]
+rpk debug bundle \
+  --upload-url <url>
+endif::[]
+
+----
++
+Example output:
++
+[.no-copy]
+----
+Creating bundle file...
+
+Debug bundle saved to "/var/lib/redpanda/1675440652-bundle.zip"
+----
+
+. On your host machine, make a directory in which to save the debug bundle:
++
+```bash
+mkdir debug-bundle
+```
+
+. Copy the debug bundle ZIP file to the `debug-bundle` directory on your host machine.
+ifdef::env-kubernetes[]
++
+Replace `<bundle-name>` with the name of your ZIP file.
++
+```bash
+kubectl cp <namespace>/redpanda-0:/var/lib/redpanda/<bundle-name> debug-bundle/<bundle-name>.zip
+```
+endif::[]
+
+. Unzip the file on your host machine and use it to debug your cluster.
++
+```bash
+cd debug-bundle
+unzip <bundle-name>.zip
+```
++
+For guidance on reading the debug bundle, see xref:troubleshoot:debug-bundle/inspect.adoc[].
+
+. Remove the debug bundle from the Redpanda broker:
++
+[,bash]
+----
+ifdef::env-kubernetes[]
+kubectl exec redpanda-0 -c redpanda --namespace <namespace> -- rm /var/lib/redpanda/<bundle-name>.zip
+endif::[]
+ifndef::env-kubernetes[]
+rm /var/lib/redpanda/<bundle-name>.zip
+endif::[]
+----
++
+TIP: To avoid manually deleting debug bundles, you can configure the xref:reference:properties/cluster-properties.adoc#debug_bundle_auto_removal_seconds[`debug_bundle_auto_removal_seconds`] property to automatically remove them after a period of time. See xref:{link-configure-bundles}#remove[Automatically remove debug bundles].
+
+When you've finished troubleshooting, remove the debug bundle from your host machine:
+
+```bash
+rm -r debug-bundle
+```
+
+[[remote-debug-bundle]]
+== Use `rpk debug remote-bundle`
+
+The xref:reference:rpk/rpk-debug/rpk-debug-remote-bundle.adoc[`rpk debug remote-bundle`] command allows you to remotely generate a consolidated debug bundle from all brokers in your cluster. This command is useful when you do not want to log into each broker's node individually using `rpk debug bundle`.
+
+. Create an xref:get-started:config-rpk-profile.adoc[`rpk` profile] to connect to your cluster. Include all brokers you want to collect data from in the `admin.hosts` configuration. For example:
```

Comment:
Since the yaml is not intended to be manipulated directly, what do you think of guiding the user on how to create one or review the current one. Something like:

```
Check the configured addresses in your profile by running `rpk profile print`:
<yaml portion they should look for>

If you want to create a new one, run `rpk profile create <profile-name> --set admin.hosts=broker1:9644,broker2:9644,...
```